docs/reference/index.asciidoc

```diff
@@ -6,41 +6,45 @@ include::links.asciidoc[]
 
 include::landing-page.asciidoc[]
 
-include::release-notes/highlights.asciidoc[]
+// overview / install
 
 include::intro.asciidoc[]
 
 include::quickstart/index.asciidoc[]
 
 include::setup.asciidoc[]
 
-include::upgrade.asciidoc[]
+// search solution
 
-include::index-modules.asciidoc[]
+include::search/search-your-data/search-your-data.asciidoc[]
 
-include::mapping.asciidoc[]
+include::reranking/index.asciidoc[]
```

Comment:
@leemthompo this is the only "lonely" topic in the search your data area. are we ok with it kinda floating like this for now?

Reply:
Yes I think we're bitten by the too deep sub-levels on the LTR pages to do otherwise. I think it's better this way than lumping LTR and semantic reranking separately into Search your data.